app/src/main/java/tech/relaycorp/ping/MainActivity.kt

```diff
@@ -67,11 +68,22 @@ class MainActivity : AppCompatActivity() {
 
         send.setOnClickListener {
             backgroundScope.launch {
+                val id = MessageId.generate()
+                val authorization = sender.issueAuthorization(
+                    recipient.identityCertificate.subjectPublicKey,
+                    ZonedDateTime.now().plusDays(3)
+                )
+                val pingMessageSerialized = serializePingMessage(
+                    id.value,
```

Comment:
FYI: We shouldn't do this in the final version of the app. We should generate a different UUID4 value.